src/cmd/report.go

```diff
@@ -73,11 +74,11 @@ type brewResp struct {
 	Items      []formulaMetric `json:"items"`
 }
 
-func fetchBrewData(formula string) {
+func reportBrewData(formula string) {
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.